### PR TITLE
Add react native example for apollo-link-state

### PR DESCRIPTION
**Pull request by amerani**

### CODE REVIEW

File: ReactNative/ApolloLinkState/App.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    newItemName: null
+  }
+
+  setNewItemName = (value) => {
+    this.setState({newItemName: value})
+  }
+
+  addItem = (store, items) => {
+    store.writeData({
+      data: { allItems: items.concat(this.state.newItemName)}
```
Comment (GantMan) [inline ReactNative/ApolloLinkState/App.js]: This needs to clear the managed component of the text input.

**Comment by GantMan** [comment]
Awwww yeah!

**Comment by GantMan** [comment]
Welcome to the contributors of the museum!

**Comment by juandc** [comment]
Congrats @amerani :smile:!

**Comment by amerani** [comment]
Thanks guys!

**Pull request merged**